### PR TITLE
Update Careers page and Navigation for functionality

### PR DESCRIPTION
**Pull request by michaelwnau**
This PR focuses on updating the /Careers/ page and the `navigation.yml` file to ensure functional navigation from the navbar. Placeholder HTML from the template has been left in place for now, but it will be replaced with dynamically served content using Jekyll in a future PR.
The changes in this PR are tied to Issue #58 , which involves improving site navigation and preparing for dynamic content serving.

Please note that this PR does not cover the customization of the careers page with markdowns and for loops. That enhancement is planned for a future PR.

### CODE REVIEW

File: careers/index.html [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                <div class="call_action_btn">
+                    <a href="#" class="zCore_btn qoute_btn_3">Openings</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<!-- End call_to_action_3 section -->
+
+{% include footer_2.html %}
```
Comment (bastosmichael) [inline careers/index.html:252]: needs new line

Comment (michaelwnau) [reply careers/index.html]: added

**Comment by michaelwnau** [comment]
Added - new line - comments in both locations.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "ansible.python.interpreterPath": "c:\\Users\\micha\\AppData\\Local\\Programs\\Python\\Python39\\python.exe"
```
Comment (bastosmichael) [inline .vscode/settings.json:2]: remove this and add the .vscode/settings.json to the .gitignore so it doesn't come back by mistake.

File: _data/navigation.yml [hunk 3]
```diff
@@ -17,7 +17,8 @@
...
 
  - text: Careers
-   url: "#"
+   url: "careers"
 
  - text: "Contact"
-   url: "contact"
\ No newline at end of file
+   url: "contact"
+   # New line added here
```
Comment (bastosmichael) [inline _data/navigation.yml:24]: please be sure to remove all missing new lines

File: careers/index.html [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+<!-- New line added here-->
```
Comment (bastosmichael) [inline careers/index.html:253]: remove missing new lines

**Review by bastosmichael — CHANGES_REQUESTED** [review]
get rid of the settings.json file that got commited since it won't apply to non windows devs and just make sure any file missing new lines (the circle and minus sign symbol) get fixed.

File: .vscode/settings.json (hunk 2 above)
Comment (michaelwnau) [reply .vscode/settings.json:2]: added to .gitignore

File: _data/navigation.yml (hunk 3 above)
Comment (michaelwnau) [reply _data/navigation.yml:24]: Deleted empty lines at 23, 24

File: careers/index.html (hunk 4 above)
Comment (michaelwnau) [reply careers/index.html:253]: Deleted empty line at 253

**Review by bastosmichael — APPROVED** [review]
Just make sure to update the title and description to better describe what you're trying to accomplish.

**Pull request merged**